.env.example

```diff
@@ -9,4 +9,5 @@ SERVICE_NAME='SERVICE_NAME'
 SERVICE_PHASE='SERVICE_PHASE'
 SERVICE_URL='SERVICE_URL'
 RATE_LIMIT_MAX=100
-RATE_WINDOW_MS=900000#15 * 60 * 1000 = 900000
\ No newline at end of file
+RATE_WINDOW_MS=900000#15 * 60 * 1000 = 900000
+API_BASE_URL='http://localhost:3000/'
```

Comment:
This is hard coding the API_BASE_URL to always use localhost. What you need to do is hook into the project hosted url. I've added a code snippet to your middleware code that does this for you.